packages/components/stories/SubHeaderBar.js

```diff
@@ -102,10 +95,16 @@ stories
 			<SubHeaderBar {...viewProps} onGoBack={backAction} />
 		</div>
 	))
-	.addWithInfo('with default left components and icon file', () => (
+	.addWithInfo('with default left components and edit', () => (
```

Comment:
Please remove all edit in all stories and add add a story to demonstrate the edit, alone `with edit button`.
The stories are like tests. If we mix them with a lot of features, it's hard to see what is its purpose.